src/Illuminate/Cache/DatabaseLock.php

```diff
@@ -94,6 +114,18 @@ protected function expiresAt()
         return $this->seconds > 0 ? time() + $this->seconds : Carbon::now()->addDays(1)->getTimestamp();
     }
 
+    /**
+     * Clear expired locks from database table, if lottery hits.
+     *
+     * @return int
+     */
+    protected function clearExpiredLocks()
```

Comment:
Defined as `@return int`, but returns nothing

Reply:
Fixed ! Thanks for catching it :)